docs/howto.md

```diff
@@ -0,0 +1,232 @@
+# How to
+
+## Add A Functional Zone
```

Comment:
I would suggest changing the terminology from "Add a Functional Zone" to "Add an item to Primary Navigation" or "Add a Primary Navigation Item"